### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by DeepDiver1975**
:tada: sabre/dav 4.0.0 is here :tada:

### CODE REVIEW

**Comment by Tntdruid** [comment]
Nice! 👍

**Comment by DeepDiver1975** [comment]
@evert @staabm please approve - I'll tag the release then asap, merge this PR - THX

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ChangeLog
 =========
 
+4.0.0 (2019-07-01)
+-------------------------
+* Release it! :shipit:
```
Comment (staabm) [inline CHANGELOG.md]: IMO we should repeat all the changes from the 4.0.0 prereleases here

**Comment by staabm** [comment]
thank you thomas for pusihing all the sabre projects. appreciate it.

File: CHANGELOG.md (hunk 1 above)
Comment (DeepDiver1975) [reply CHANGELOG.md]: ok - will update ...

**Comment by DeepDiver1975** [comment]
@staabm changelog has been updated

**Comment by DeepDiver1975** [comment]
Here is the tag: https://github.com/sabre-io/dav/releases/tag/4.0.0

**Comment by evert** [comment]
Congrats everyone. It's pretty great to see this =)

**Pull request merged**